internal/file/zip_file_helpers_test.go

```diff
@@ -73,7 +72,7 @@ func assertNoError(t testing.TB, fn func() error) func() {
 func setupZipFileTest(t testing.TB, sourceDirPath string, zip64 bool) string {
 	t.Helper()
 
-	archivePrefix, err := ioutil.TempFile("", "syft-ziputil-archive-TEST-")
+	archivePrefix, err := os.CreateTemp("", "syft-ziputil-archive-TEST-")
```

Comment:
I believe these `ioutil.TempFile` calls can be replaced with `path.Join(t.TempDir(), <start-of-file-name>)`, e.g.:
```
archivePrefix := path.Join(t.TempDir(), "syft-ziputil-archive-TEST-")
```